### ISSUE TITLE
"Find out more about the Xamarin Workbooks Experience Improvement Program" hyperlink is not accessible using keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#263]
### Steps to Reproduce

1. Launch Xamarin Workbook application and select Mac platform and create a new workbook.
2. Navigate to Xamarin Workbooks-> preferences->Hit On Enter key-> General tab is opened by default.
3. Navigate to Feedback tab-> Hit on Enter.
4. Feedback tab is opened.
5. Try to get focus on the below hyperlink using keyboard:
"Find out more about the Xamarin Workbooks Experience Improvement Program" Hyperlink.

### Expected Behavior
Hyperlink should be accessible and invoked using keyboard.

### Actual Behavior
The below  hyperlink is not accessible using keyboard.
Hyperlink : "Find out more about the Xamarin Workbooks Experience Improvement Program."

### User Impact:
The keyboard user won't be able to interact with this hyperlinks.

### Environment

MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)

### Log File

![kb2_feedback_hyperlink](https://user-images.githubusercontent.com/22956130/39856893-134c4696-5421-11e8-952a-f856d8474e50.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS2.1.1
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1819
